src/recipe/stage2/build.rs

```diff
@@ -0,0 +1,706 @@
+            }
+            Ok(secrets)
+        }
+        Node::Mapping(_) => Err(_partialerror!(
+            *node.span(),
+            ErrorKind::Other,
+            label = "expected scalar or sequence"
+        )),
+    }
+}
+
+fn parse_passthrough(node: &Node, jinja: &Jinja) -> Result<Vec<String>, PartialParsingError> {
+    match node {
+        Node::Scalar(s) => {
+            let passthrough = jinja.render_str(s.as_str()).map_err(|err| {
+                _partialerror!(
+                    *s.span(),
+                    ErrorKind::JinjaRendering(err),
+                    label = "error rendering `passthrough`"
+                )
+            })?;
+
+            if passthrough.is_empty() {
+                Err(_partialerror!(
+                    *s.span(),
+                    ErrorKind::Other,
+                    label = "empty string is not allowed in `passthrough`"
+                ))
+            } else {
+                Ok(vec![passthrough])
+            }
+        }
+        Node::Sequence(seq) => {
+            let mut passthrough = Vec::new();
+            for inner in seq.iter() {
+                match inner {
+                    SequenceNodeInternal::Simple(n) => {
+                        passthrough.extend(parse_passthrough(n, jinja)?)
+                    }
+                    SequenceNodeInternal::Conditional(if_sel) => {
+                        let if_res = if_sel.process(jinja)?;
+                        if let Some(if_res) = if_res {
+                            passthrough.extend(parse_passthrough(&if_res, jinja)?)
+                        }
+                    }
+                }
+            }
+            Ok(passthrough)
+        }
+        Node::Mapping(_) => Err(_partialerror!(
+            *node.span(),
+            ErrorKind::Other,
+            label = "expected scalar or sequence"
+        )),
+    }
+}
+
+/// Run exports are applied to downstream packages that depend on this package.
+#[derive(Debug, Default, Clone, Serialize)]
+pub struct RunExports {
+    /// Noarch run exports are the only ones looked at when building noarch packages.
+    pub(super) noarch: Vec<Dependency>,
+    /// Strong run exports apply from the build and host env to the run env.
+    pub(super) strong: Vec<Dependency>,
+    /// Strong run constrains add run_constrains from the build and host env.
+    pub(super) strong_constrains: Vec<Dependency>,
+    /// Weak run exports apply from the host env to the run env.
+    pub(super) weak: Vec<Dependency>,
+    /// Weak run constrains add run_constrains from the host env.
+    pub(super) weak_constrains: Vec<Dependency>,
+}
+
+impl RunExports {
+    fn from_node(node: &Node, jinja: &Jinja) -> Result<RunExports, PartialParsingError> {
+        let mut run_exports = RunExports::default();
+
+        match node {
+            Node::Scalar(_) | Node::Sequence(_) => {
+                let deps = parse_dependency(node, jinja)?;
+                run_exports.strong = deps;
```

Comment:
This should be a weak run export by default (cf. conda-build https://github.com/conda/conda-build/blob/dda39190e86efb1b6632ce8bf4fb31393ef8785e/conda_build/build.py#L1517-L1518)